### PR TITLE
New package: gflags-2.1.2

### PR DESCRIPTION
**Pull request by diogoleal**
The directory doc/ contains a html and css files, that is why I install it's files.

### CODE REVIEW

File: srcpkgs/gflags/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+build_style=cmake
+hostmakedepends="cmake"
+short_desc="A C++ library that implements commandline flags processing"
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="Google"
+homepage="http://gflags.github.io/gflags"
+distfiles="https://github.com/gflags/gflags/archive/v${version}.tar.gz"
+checksum=d8331bd0f7367c8afd5fcb5f5e85e96868a00fd24b7276fa5fcee1e5575c2662
+
+do_install(){
```
Comment (xtraeme) [inline srcpkgs/gflags/template]: `do_install` -> `post_install`

File: srcpkgs/gflags/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	vlicense COPYING.txt
+	vdoc ChangeLog.txt
+	vdoc README.md
+}
```
Comment (xtraeme) [inline srcpkgs/gflags/template]: Please add a newline.

**Comment by diogoleal** [comment]
Done

File: srcpkgs/gflags/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'gflags'
+pkgname=gflags
+version=2.1.2
+revision=1
+build_style=cmake
+hostmakedepends="cmake"
+short_desc="A C++ library that implements commandline flags processing"
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="Google"
```
Comment (xtraeme) [inline srcpkgs/gflags/template]: The license is not `Google` it's a standard `3-clause-BSD` license.

Comment (diogoleal) [inline srcpkgs/gflags/template]: changed.


Diogo Leal [estranho]
Site : http://diogoleal.com
Nostalgia Games: http://nostalgiagames.com.br
talk: estranho@jabber-br.org

**Pull request merged**